hedera-node/hedera-app/src/main/java/com/hedera/node/app/ids/schemas/V0590EntityIdSchema.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2024-2025 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.ids.schemas;
+
+import com.hedera.hapi.node.base.SemanticVersion;
+import com.hedera.hapi.node.state.entity.EntityCounts;
+import com.swirlds.state.lifecycle.MigrationContext;
+import com.swirlds.state.lifecycle.Schema;
+import com.swirlds.state.lifecycle.StateDefinition;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.Set;
+
+public class V0590EntityIdSchema extends Schema {
```

Comment:
Should it be named V059EntityCountersSchema instead of V059EntityIdSchema? There is alread "entity ID schema" in 0.49

Reply:
Usually the Schemas are named with Service Name and version it will be applied from. Since the Service name is `EntityIdService` I named this schema as `V059EntityIdSchema`.  The `V049EntityidSchema` applies the new states and migration that needs to be done in that version.